docs/networks.rst

```diff
@@ -10,7 +10,8 @@ though usually not `.Probe` instances.)
 Like most abstractions, this helps with code-reuse and maintainability.
 You'll find the documentation for the various pre-built networks below.
 
-Building your own network can be a great way to encapsulate
+You may also want to pre-build your own networks.  Doing so can
```

Comment:
The concept of "pre-building" a network feels strange to me, which makes me think that "pre-built" networks might be confusing too.

How about instead of "pre-built" we introduce the term "Reusable networks"; then, it makes more sense to tell users that they can also build reusable networks (rather the pre-built networks).

Reply:
yeah, i wasn't a fan of 'pre-built' either (mostly when referring to ones users would make). '  'Reusable' is better.  I've been trying to think of something even better. Have come up with 'pre-built', 'template', 'component', but none of those sound any better, so 'reusable' is fine by me.